packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -404,9 +404,10 @@ class TooltipState extends State<Tooltip> with SingleTickerProviderStateMixin {
     final ThemeData theme = Theme.of(context);
     switch (theme.platform) {
       case TargetPlatform.macOS:
```

Comment:
The original issue also mentioned macOS, am i missing somethig?

> This legibility issue also applies to Linux and MacOS platforms when device pixel ratio 1.0 is used.

Reply:
My bad, I read all the comments on the previous issue #71052 that mainly mentioned Windows and Linux, and I didn't notice that MacOS is included in the new issue description. Good catch! That's make this change even simpler.